### PR TITLE
Fix zhanqi

### PR DESCRIPTION
**Pull request by liushuyu**
Zhanqi extractor works again...
And added ts merging implementation...

Signed-off-by: liushuyu <liushuyu_011@163.com>

[Review on Reviewable](https://reviewable.io/reviews/soimort/you-get/546)

### CODE REVIEW

**Comment by jackyzy823** [comment]
some urls for vods return 404 in zhaqi.py Ln.41  such as `http://www.zhanqi.tv/videos/dota2/2015/06/23670.html`.

**Comment by liushuyu** [comment]
Oh ,f**k
Zhanqi updated its server again!!!
Maybe I need to change...

**Comment by liushuyu** [comment]
@jackyzy823 I fixed this again! Why not giving a try?

**Comment by liushuyu** [comment]
Zhanqi will split video to hundred of thousands of video parts, it will be very hard (slow) to get the real size.
And if you make users wait so long, users will think 'Oh, no responding...' 
So I really can't think up a solution...

File: src/you_get/extractors/zhanqi.py [hunk 1]
```diff
@@ -7,22 +7,46 @@
...
+        vod_m3u8 = get_html(vod_m3u8_request)
+        part_url = re.findall(r'(/[^#]+)\.ts',vod_m3u8)
+        real_url = []
+        for i in part_url:
+            i = vod_base + i + ".ts"
+            real_url.append(i)
+        type_ = ''
+        size = 0
+        for url in real_url:
+        _, type_, temp = url_info(url)
```
Comment (soimort) [inline src/you_get/extractors/zhanqi.py]: Have you actually run this part of code? It's an `IndentationError` here.

**Comment by liushuyu** [comment]
I am so sorry for that.
I had a fever last night, and I can't remember what I have done...

File: src/you_get/extractors/zhanqi.py (hunk 1 above)
Comment (liushuyu) [inline src/you_get/extractors/zhanqi.py]: Fixed ....

**Comment by soimort** [comment]
Never mind, everybody makes mistakes.
Thanks for the PR.

**Pull request merged**